services/tenants/dashboard/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
```

Comment:
this whole file can be ignored since we're gonna merge this into the tenants service